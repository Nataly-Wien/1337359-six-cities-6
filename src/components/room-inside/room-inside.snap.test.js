import React from 'react';
import {render} from '@testing-library/react';
import RoomInside from './room-inside';

const goods = [`Air conditioning`, `Dishwasher`, `Flat-screen TV`, `Coffee machine`, `Free WiFi`];

it(`Should room facilities render correctly`, () => {
  const {container} = render(<RoomInside goods={goods} />);
  expect(container).toMatchSnapshot();
});
