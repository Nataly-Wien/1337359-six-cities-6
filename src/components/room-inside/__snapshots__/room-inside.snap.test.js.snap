// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should room facilities render correctly 1`] = `
<div>
  <div
    class="property__inside"
  >
    <h2
      class="property__inside-title"
    >
      What's inside
    </h2>
    <ul
      class="property__inside-list"
    >
      <li
        class="property__inside-item"
      >
        Air conditioning
      </li>
      <li
        class="property__inside-item"
      >
        Dishwasher
      </li>
      <li
        class="property__inside-item"
      >
        Flat-screen TV
      </li>
      <li
        class="property__inside-item"
      >
        Coffee machine
      </li>
      <li
        class="property__inside-item"
      >
        Free WiFi
      </li>
      ;
    </ul>
  </div>
</div>
`;
